### DISCUSSION TITLE
List snippets

### DISCUSSION THREAD

**Discussion by smarquez1** [ibhagwan/fzf-lua#1616] [Q&A]
I'm trying to create a mapping to list snippets without success.

Is there any way to do that? I'm currently using native `vim.snippet` (no external libraries like LuaSnip)?

This is useful to me to learn snippets of a new language, and to verify that my custom snippet list is being loaded too.

Thanks!

**Comment by ibhagwan** [comment]
See the `fzf_exec` API, it’s very simple to list anything, especially lua tables:
https://github.com/ibhagwan/fzf-lua/wiki/Advanced

I’m not experienced with snippets but I’m sure there’s a way to enumerate available snippets using `vim.snippet`, find out how to do that and feed it into fzf_exec in the required format (simple table of strings ).

**Reply by smarquez1** [reply]
Thanks! I'll try to find a way to do this and report back

**Reply by smarquez1** [reply]
I found a gist to list snippets for different libraries. I guess I need to use one such as Luasnip to achieve this for now, and will try to find a way to do it using native `vim.snippet` https://gist.github.com/qanyue/acc75f112489731c28e9893d6e18366e
